modules/rtbhouseBidAdapter.js

```diff
@@ -96,7 +96,8 @@ function mapImpression(slot) {
     id: slot.bidId,
     banner: mapBanner(slot),
     native: mapNative(slot),
-    tagid: slot.adUnitCode.toString()
+    tagid: slot.adUnitCode.toString(),
+    bidfloor: slot.params.bidfloor
```

Comment:
tryparse int/float maybe? There might be something riddiculously unexpected, needs sanitization imho

Reply:
Good point, fixed.